guides/common/modules/proc_setting-the-provisioning-context-by-using-web-ui.adoc

```diff
@@ -11,17 +12,5 @@ If you have not selected an organization and location to use, the menu displays:
 .Procedure
 . Click *Any Organization* and select the organization.
 . Click *Any Location* and select the location to use.
-
-Each user can set their default provisioning context in their account settings.
+. Each user can set their default provisioning context in their account settings.
```

Comment:
Unsure if this is really a step or more of a "next step"? Your call.

Reply:
Definitely not a Next steps section, those are only for links and not for actual steps.
https://redhat-documentation.github.io/modular-docs/#next-steps